spec/integration/active_record_spec.rb

```diff
@@ -25,7 +25,6 @@ def restart_dummy_rails_server
 
   after(:all) do
     stop_rails_server
-    teardown_rails_server
```

Comment:
This was supposed to drop the test database but there is no need for us to do so. We will now recreate the test database at the start of each test.